integration_test/third_party_apps_test/applications/nvml/metadata.yaml

```diff
@@ -45,6 +45,7 @@ gpu_platforms:
       - centos-cloud:centos-7
       - debian-cloud:debian-10
       - debian-cloud:debian-11
+      - ml-images:common-gpu-debian-11-py310
```

Comment:
So it broke with nvml but not dcgm?

Reply:
This is adding this as a platform to test on and the tests look like it passed

Reply:
Erm, sorry, misread the code. [Optional] Should we add it for dcgm as well?

Reply:
good call. added it.